tree/dataframe/test/dataframe_helpers.cxx

```diff
@@ -324,6 +324,91 @@ TEST(RDFHelpers, SaveGraphSharedDefines)
    EXPECT_EQ(graph, expected);
 }
 
+TEST(RDFHelpers, GraphAsymmErrorsContainers)
+{
+   std::vector<double> xx = {-0.22, 0.05, 0.25, 0.35, 0.5, 0.61, 0.7, 0.85, 0.89, 0.95};
+   std::vector<double> yy = {1., 2.9, 5.6, 7.4, 9, 9.6, 8.7, 6.3, 4.5, 1};
+   std::vector<double> exl = {.05, .1, .07, .07, .04, .05, .06, .07, .08, .05};
+   std::vector<double> exh = {.02, .08, .05, .05, .03, .03, .04, .05, .06, .03};
+   std::vector<double> eyl = {.8, .7, .6, .5, .4, .4, .5, .6, .7, .8};
+   std::vector<double> eyh = {.6, .5, .4, .3, .2, .2, .3, .4, .5, .6};
+
+   auto df = RDataFrame(3)
```

Comment:
we are creating 3 entries but only checking the values from the first. so let's create just 1 entry instead:

```suggestion
   auto df = RDataFrame(1)
```